src/formio/components/Map.js

```diff
@@ -78,8 +78,10 @@ export default class Map extends Field {
       mapContainer: 'single',
     });
     return super.attach(element).then(() => {
-      this.reactRoot = createRoot(this.refs.mapContainer);
-      this.renderReact();
+      if (this.refs.mapContainer) {
+        this.reactRoot = createRoot(this.refs.mapContainer);
+        this.renderReact();
```

Comment:
I don't understand how this line is not covered when there's a test?

Reply:
Yes I don't understand as well, and adding a `console.log` in this code path confirms it is covered by the story by running it in the browser

Reply:
hrm, I suspect JS race conditions and instrumentation not playing along well then